sr_port/have_crit.h

```diff
@@ -84,14 +83,42 @@ typedef enum
 } intrpt_state_t;
 
 GBLREF	intrpt_state_t	intrpt_ok_state;
-GBLREF	boolean_t	deferred_timers_check_needed;
+
+GBLREF	boolean_t	deferred_signal_handling_needed; /* a bitmask of the below DEFERRED_SIGNAL_HANDLING_NEEDED_* macros */
```

Comment:
A boolean type is really only supposed to be a single bit (true or false). Since this field has multiple bit flags in it, it should be a uint4 type critter I think.

Reply:
I thought about that but used an existing usage TREF(donot_commit). I will fix this post-r1.22 assuming I don't need to change r1.22 again for some other issue.

Reply:
This is addressed in #249